.changeset/nervous-games-reflect.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react-brand': minor
```

Comment:
Could we make this a patch? Usually use minor to indicate there's major (as we're pre-v1) or new components